lib/hanami/config/logger.rb

```diff
@@ -89,8 +120,16 @@ def app_name(value = nil)
       # @see Hanami::Config::Logger#stream
       # @see Hanami::Config::Logger#engine
       def build
-        @engine ||
-          ::Hanami::Logger.new(@app_name, device: @device)
+        logger = ::Hanami::Logger.new(@app_name, device: @device)
```

Comment:
@davydovanton Do you think it's the case we're gonna modify the initializer so we can achieve the following code?

```ruby
@engine || ::Hanami::Logger.new(@app_name, device: @device, level: @level)
```

More elegant, doesn't instantiate an object that can theoreticallly not be used, and avoids mutations of the logger after it's being instantiated. It looks like a win-win situation.